portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -0,0 +1,97 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.Query;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+/** Servlet that returns comments */
+@WebServlet("/comment")
+public class CommentServlet extends HttpServlet {
+
+  /** Comment data structure */
+  protected class Comment{
+    private final String text;
+    private final String user;
+    private final long timestamp;
+
+    public Comment(String user, String text, long timestamp){
+      this.text = text;
+      this.user = user;
+      this.timestamp = timestamp;
+    }
+
+  }
+
+  /** Responds a JSON with all the comments in DataStore in descending chronological order */
```

Comment:
Nit: I suggest changing this wording to something like "Adds all the comments in the DataStore to the response argument, in descending chronological order" It's a bit more clear that this function doesn't do this by returning.

Reply:
Resolved, thanks for pointing that!